scripts/dev/count_lines_of_code

```diff
@@ -176,7 +177,7 @@ def PrintCounters(arr, eArr, total, eTot):
          PrintRow([ft] + vals + [GetEmptyPercStr(vals[0], vals[1])])
 
    print(("-" * 50))
-   PrintRow(["Total:", total, eTot, GetEmptyPercStr(total, eTot)])
+   PrintRow(["Total:", str(total), str(eTot), GetEmptyPercStr(total, eTot)])
```

Comment:
If you make `PrintRow` to accept as `List[Any]`, you won't need to do this extra conversions to string here.